dist/tools/cosy/patches/0001-cosy.py-set-working-directory-to-location-of-cosy.py.patch

```diff
@@ -1,4 +1,4 @@
-From 77f489ec59d30c89e41bb94e2094b8bbaeeec4fb Mon Sep 17 00:00:00 2001
+From d19729598e557dcac364b3d80f2ee2f16fc2de08 Mon Sep 17 00:00:00 2001
```

Comment:
I think we can drop this patch too

Reply:
Thanks. Missed that because the merged resolution was way different than the approach taken here. Removed, and indeed still works.